AK/Vector.h

```diff
@@ -182,21 +184,21 @@ requires(!IsRvalueReference<T>) class Vector {
         return TypedTransfer<StorageType>::compare(data(), other.data(), size());
     }
 
-    bool contains_slow(T const& value) const
+    bool contains_slow(VisibleType const& value) const
     {
         for (size_t i = 0; i < size(); ++i) {
-            if (Traits<T>::equals(at(i), value))
+            if (Traits<VisibleType>::equals(at(i), value))
```

Comment:
Note that this changes the semantics from comparing references to comparing the values, these two currently do the same thing, so this change is fine (theoretically, if someone doesn't override `Traits<Foo&>`)

Reply:
Yes, also note the `at(i)` and `value` are now `const&` and not `&` anymore, which makes sense because we shouldn't need mutable references to compare two values.